genesyscloud/resource_genesyscloud_architect_emergencygroup_test.go

```diff
@@ -22,27 +22,30 @@ func TestAccResourceArchitectEmergencyGroups(t *testing.T) {
 
 		updatedDescription = description + " updated"
 
-		ivrResourceID = "test-ivr"
-		ivrName       = "Test IVR " + uuid.NewString()
-
 		flowResource      = "test_flow"
 		flowName          = "Terraform Test Flow " + uuid.NewString()
 		flowFilePath      = "../examples/resources/genesyscloud_flow/inboundcall_flow_example.yaml"
 		inboundCallConfig = fmt.Sprintf("inboundCall:\n  name: %s\n  defaultLanguage: en-us\n  startUpRef: ./menus/menu[mainMenu]\n  initialGreeting:\n    tts: Archy says hi!!!\n  menus:\n    - menu:\n        name: Main Menu\n        audio:\n          tts: You are at the Main Menu, press 9 to disconnect.\n        refId: mainMenu\n        choices:\n          - menuDisconnect:\n              name: Disconnect\n              dtmf: digit_9", flowName)
 	)
 
+	config, err := AuthorizeSdk()
+	if err != nil {
+		t.Skip("failed to authorize client credentials")
+	}
+
+	// TODO: Create the IVR inside the test config once emergency group has been moved to its own package.
+	// Currently, the ivr resource cannot be registered for these tests because of a cyclic dependency issue.
+	ivrId := "f94e084e-40eb-470b-80d6-0f99cf22d102"
+	if !ivrExists(config, ivrId) {
+		t.Skip("Skipping because IVR does not exists in the target org.")
+	}
```

Comment:
The architect IVR resource was being registered for this emergency group test. That cannot be done now because a cyclic dependency would exist between the `genesyscloud` and `architect_ivr` package. As a temporary workaround, I left an IVR in the inintesting org and hardcoded the ID here. I could create the IVR using the go sdk but I would also have to create an associated flow, which would be more complex.